src/content/docs/telemetry-data-platform/ingest-manage-data/manage-data/query-limits.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: Query system limits
+tags:
+  - Telemetry Data Platform
+  - Ingest data manage data
+  - Manage data
+  - Resource metrics
+  - system limits
+
+---
+
+As you use the New Relic system there are resource limits in place to protect your experience, our systems, and our other customers. These limits range from the maximum number of characters you can have in a query, to API request rates, to how many events your queries inspect. And everywhere in between.
+
+This page describes the limit metrics and NrIntegration events that enable you to view  your limits, your current data usage and overall resource consumption as compared to those limits, and the impact of experiencing a limit event. We also provide a handful of queries that, when compiled into a dashboard, can give you consistent insight into your limits status.
+
+<Callout variant="important">
+While `NrIntegration` events cover nearly all limits, resource metrics currently only cover request rate ingestion limits.
+</Callout>
+
+## What happens when you reach a limit 
+
+Our response to reaching a limit depends on a handful of factors: the [type of limit that’s reached](/docs/telemetry-data-platform/ingest-manage-data/manage-data/view-system-limits/#violations), as well as the duration, frequency, and amount at which you exceed the limit.  Exceeding a limit doesn’t always mean you experience a limit event, such as dropped data, rejected traffic, or having your data turned off for the rest of the day. We sometimes allow a small buffer before enforcing a limit. That said, any resource consumed above 100% is at risk for limit impact at any time.
+
+Many of our rate limits apply proportionally. That means if you’re barely exceeding the limit, we will take less action than if you're exceeding by 200%.
+
+Limit status is only visible if you're sending data in to a corresponding `dataType` or `limitName` API. For example, if you send in data via the Metric API, you’ll see the Metric API resource metrics, but if you don't send any APM data in, you won't see APM resource metrics.
```

Comment:
`limit metrics are` instead of `limit status is`

I am not sure what limit status means